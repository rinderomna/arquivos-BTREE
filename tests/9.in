7 tipo1 binario9.bin indice9.bin 1
935 2004 56 "PR" "SAO JOSE DOS PINHAIS" "HONDA" NULO
