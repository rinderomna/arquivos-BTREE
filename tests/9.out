MARCA DO VEICULO: NAO PREENCHIDO
MODELO DO VEICULO: CG 125 TITAN
ANO DE FABRICACAO: 1997
NOME DA CIDADE: ANTONIO OLINTO
QUANTIDADE DE VEICULOS: 10

